File: pgm/ABC_C#/ABC_172_B.cs
using System;
using System.Collections.Generic;
using System.Linq;
// using System.Text;

class Program
{
    static void Main()
    {
        string S = Console.ReadLine();
        string T = Console.ReadLine();
        var z = S.Zip(T, (first, second) => new char[] { first, second });
        int ans = 0;
        foreach (var v in z) if (v[0] != v[1]) ans++;
        Console.WriteLine(ans);
        return;
    }
    IEnumerable<TResult> Zip<TResult, T1, T2>
        (IEnumerable<T1> a,
         IEnumerable<T2> b,
         Func<T1, T2, TResult> combine)
    {
        using (var f = a.GetEnumerator())
        using (var s = b.GetEnumerator())
        {
            while (f.MoveNext() && s.MoveNext())
                yield return combine(f.Current, s.Current);
        }
    }
}

File: pgm/ABC_Visual_Basic/ABC_172_B.vb
' Imports System.Collections.Generic
' Imports System.Linq
' Imports System.Text
' Imports System.Text.RegularExpressions

Class Main
    Public Shared Sub Main()
        Dim S As String = Console.ReadLine()
        Dim T As String = Console.ReadLine()
        Dim z = S.Zip(T, Function(first, second) New Char() { first, second })
        Dim ans = 0
        For Each v in z
            If v(0) <> v(1) Then
                ans += 1
            End If
        Next
        Console.WriteLine(ans)
        Return
    End Sub
End Class